editor/plugins/visual_shader_editor_plugin.cpp

```diff
@@ -376,6 +465,16 @@ void VisualShaderGraphPlugin::set_frame_autoshrink_enabled(VisualShader::Type p_
 	frame->set_autoshrink_enabled(p_enable);
 }
 
+void VisualShaderGraphPlugin::update_reroute_nodes() {
+	for (const KeyValue<int, Link> &E : links) {
+		Ref<VisualShaderNodeReroute> reroute_node = Object::cast_to<VisualShaderNodeReroute>(E.value.visual_node);
+		if (reroute_node.is_null()) {
+			continue;
+		}
+		update_node(visual_shader->get_shader_type(), E.key);
```

Comment:
```suggestion
		if (reroute_node.is_valid()) {
			update_node(visual_shader->get_shader_type(), E.key);
		}
```
In this case early continue is not very beneficial.